contracts/src/components.cairo

```diff
@@ -36,10 +37,22 @@ enum Roles {
 impl RolesSerdeLen of dojo::SerdeLen<Roles> {
     #[inline(always)]
     fn len() -> usize {
-        1
+        2
```

Comment:
revert this, the size is 1

Reply:
[review fix](https://github.com/keep-starknet-strange/tsubasa/pull/69/commits/a1ee4350e0b92ada5d02744875b6b08aee6adecb)